### PR TITLE
Using executor in cv pipeline preprocessing

### PR DESCRIPTION
**Pull request by corey-nm**
As part of #604, preprocessing was moved out of async threadpool execution. This PR changes the CV pipelines (image classification, yolo, yolact), to use `Pipeline.executor` in their preprocessing scripts. This is a performance improvement for static batch, because previously async preprocessing was not done for static batch.

Note: This only makes a difference when the pipelines need to load images. If pipelines receive all numpy arrays, then there is no performance gain when using async preprocessing.

### CODE REVIEW

**Comment by corey-nm** [comment]
Note: this is blocked by #604 because that PR makes it so executor always exists. Currently on main it's possible executor is None

**Comment by bfineran** [comment]
looks like the rebase should also unblock the tests

**Review by rahul-tuli — COMMENTED** [review]
Lgtm pending rebase!

**Review by KSGulin — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**
